src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx

```diff
@@ -307,13 +311,14 @@ export const VenueRoomsEditor: React.FC<VenueRoomsEditorProps> = ({
       {!backgroundImage ? (
         <MapBackgroundPlaceholder />
       ) : (
-        <img
-          alt="draggable background "
-          className={`Container__background-image ${backgroundImageClassName}`}
-          src={backgroundImage}
-        />
+        <div className="VenueRoomsEditor__background-image-wrapper">
```

Comment:
Do you actually needs this div? What was wrong with the img?

Reply:
@denisdimitrov it's impossible to display the preview without div